### DISCUSSION TITLE
Is there a way to clear the focus ?

### DISCUSSION THREAD

**Discussion by maxigit** [qutebrowser/qutebrowser#7680] [Q&A] [answered]
I'm not sure the term is correct but , when you load a page no element are in focus. If you click into a text input for example this element become in focus and you can navigate to the next and previous elemest usin Tab Shift/Tab. However sometime I need to go back to no focus at all (or maybe focus to the parent ?) so that I can use some keyboard key which otherwise are trapped by the focus element. Is there a way to do so (pressing Esc go back to normal mode but doesn't change the focus).

**Comment by The-Compiler** [comment, marked as answer]
What keyboard keys are trapped by it?

`:jseval -q document.activeElement.blur()` might be what you want.

**Comment by maxigit** [comment]
Thanks for you prompt answer !
J and K  (and SPC) are trapped for example on dropdown lists and J and K
are trapped on text input (the cursor goes to the beginning or back of the
input)
So when you start filling a form with lots of text inputs there is no way
to scroll down using J and K (to show the submit button for example)
without either tabbing ALL the fields
or using the mouse to click somewhere else.

What you proposed seems to work, I'll try to map it to ESC (I think it
should be the default)
